dali/base/dadfs.hpp

```diff
@@ -814,6 +814,10 @@ inline bool isPartTLK(IPropertyTree &pt) { const char *kind = pt.queryProp("@kin
 inline bool isPartTLK(IDistributedFilePart *p) { return isPartTLK(p->queryAttributes()); }
 inline bool isPartTLK(IPartDescriptor *p) { return isPartTLK(p->queryProperties()); }
 
+inline const char *getFileKind(IPropertyTree &pt) { return pt.queryProp("@kind"); }
```

Comment:
picky, but queryFileKind might be more appropriate since the caller does not take ownership of the returned string.

Reply:
fair enough, I will change it.